terraform/task_definition.tf

```diff
@@ -0,0 +1,21 @@
+resource "aws_ecs_task_definition" "task_definition" {
```

Comment:
And this file to backend as well. So whenever we build a new image, we can update our `task_deifinition`